x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/osquery/not_available_prompt.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCode, EuiEmptyPrompt } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { i18n } from '@kbn/i18n';
+
+export const PERMISSION_DENIED = i18n.translate(
+  'xpack.securitySolution.markdown.osquery.permissionDenied',
+  {
+    defaultMessage: 'Permission denied',
+  }
+);
+
+export const OsqueryNotAvailablePrompt = () => (
+  <EuiEmptyPrompt
+    iconType="logoOsquery"
+    title={<h2>{PERMISSION_DENIED}</h2>}
+    titleSize="xs"
+    body={
+      <p>
+        <FormattedMessage
+          id="xpack.securitySolution.markdown.osquery.missingPrivilleges"
+          defaultMessage="To access this page, ask your administrator for {osquery} Kibana privileges."
+          values={{
+            // eslint-disable-next-line react/jsx-no-literals
+            osquery: <EuiCode>osquery</EuiCode>,
```

Comment:
I think the kibana pattern is `<EuiCode>{'osquery'}</EuiCode>` so you wouldn't need to disable the line, though I understand the fact that it's not really needed here.

Reply:
Thank you! Let me check, that would a game changer for me! :D

Reply:
Indeed it works! Thank you :)